### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by alneberg**
Updated the test_integration.py:
 - Now works with python 3.
 - Avoids some problematic URLs
 - Removed calls to outdated URLs (mainly to the old flowcells_db and the old samples_db)

Also removed api URL:s that were only fetching from the samples database (it has not been updated for 4 years now).

### CODE REVIEW

File: status/queues.py [hunk 1]
```diff
@@ -48,7 +48,10 @@ def get(self):
                     library_type = ''
                     runmode = ''
                     if not 'lambda DNA' in art.name:
-                        library_type = art.samples[0].project.udf["Library construction method"]
+                        try:
```
Comment (aanil) [inline status/queues.py:51]: Maybe `get` instead of `try` `except` ?

Comment (alneberg) [reply status/queues.py]: Good idea. I've changed it now.

**Review by aanil — APPROVED** [review]
Looks good!

File: status/queues.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def get(self):
                     library_type = ''
                     runmode = ''
                     if not 'lambda DNA' in art.name:
-                        library_type = art.samples[0].project.udf["Library construction method"]
+                        library_type = art.samples[0].project.udf.get("Library construction method", 'NA')
                         try:
```
Comment (ssjunnebo) [inline status/queues.py:52]: I guess you could use get here as well?

File: tests/test_integration.py [hunk 3]
```diff
@@ -8,22 +8,22 @@
...
+TEST_ITEMS = os.path.join(file_dir_path, "test_items.yaml")
 
 class TestGet(object):
     def setUp(self):
         """Server Settings"""
-        self.url = 'http://localhost:8889'
+        self.url = 'http://localhost:9761'
         self.api = requests.get(self.url + '/api/v1')
-        assert_true(self.api.ok)
+        #assert_true(self.api.ok)
```
Comment (ssjunnebo) [inline tests/test_integration.py:18]: Should this be kept?

**Review by ssjunnebo — APPROVED** [review]
Looks good! 👍 Ready to merge when you are 🙂

File: status/queues.py (hunk 2 above)
Comment (aanil) [reply status/queues.py:52]: Maybe next time? The code hasn't really changed here?

File: tests/test_integration.py (hunk 3 above)
Comment (alneberg) [reply tests/test_integration.py:18]: Good catch. Should probably add that back into action!

File: status/queues.py (hunk 2 above)
Comment (alneberg) [reply status/queues.py:52]: I was thinking the same as both of you when I adjusted the code according to Anands recommendation. I'll just keep it as it is for now.

**Comment by alneberg** [comment]
> Looks good! 👍 Ready to merge when you are 🙂

Ready now. Thanks!

**Pull request merged**